fe/fe-core/src/main/java/com/starrocks/analysis/FunctionCallExpr.java

```diff
@@ -459,4 +460,14 @@ public boolean isSelfMonotonic() {
         }
         return false;
     }
+
+    @Override
+    public Expr uncheckedCastTo(Type targetType) throws AnalysisException {
```

Comment:
why need override this method?

Reply:
Because it add CastExpr in Expr, But CastExpr should be added only when type is not equals.